sdk/servicebus/azure-messaging-servicebus/src/main/java/module-info.java

```diff
@@ -10,6 +10,7 @@
     exports com.azure.messaging.servicebus.models;
 
     opens com.azure.messaging.servicebus.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
```

Comment:
Are there still any classes in implementation.models since we moved it to administration? if not, we could remove this line.